### PR TITLE
Tint no upload icon grey

### LINKED ISSUE #2478: Upload view: "no uploads" not tinted?
Shall this upload icon be tinted? For me it looks kind of weird as the default blue is never shown everywhere else.

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #2478 
I chose to keep the svg file, in case we want to edit something, although svg and xml are nearly the same
(damn autoformatter on svg file).

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/main/res/drawable/ic_list_empty_upload.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        android:viewportWidth="24"
+        android:viewportHeight="24">
+
+    <path
+        android:fillColor="#888888"
+        android:fillAlpha="0.5"
+        android:strokeAlpha="0.5"
+        android:strokeWidth="1"
+        android:pathData="M 9,16 V 10 H 5 l 7,-7 7,7 h -4 v 6 H 9 m -4,4 v -2 h 14 v 2 z"/>
+</vector>
```
Comment (mario) [inline src/main/res/drawable/ic_list_empty_upload.xml]: New line.

Comment (AndyScherzinger) [reply src/main/res/drawable/ic_list_empty_upload.xml]: Fixed + rebase: https://github.com/nextcloud/android/pull/2489/commits/e3bf29b4b4b0f8b050df3b776a9f7deacb74b955 for approval

**Comment by AndyScherzinger** [comment]
:+1: fine by me

**Comment by mario** [comment]
👍

**Pull request merged**